Gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-gem "solidus", git: "git@github.com:solidusio/solidus.git", branch: "master"
+gem "solidus", git: "git@github.com:athal7/solidus.git", branch: "allow-api-key-generation-without-persistence"
```

Comment:
note to change this back after/if https://github.com/solidusio/solidus/pull/396 has been accepted